scripts/prometheus_on_kind.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+#
+# Copyright (c) 2020, 2021 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# include the common_utils.sh script to access methods
+current_dir="$(dirname "$0")"
+source ${current_dir}/common_utils.sh
+non_interactive=0
+# Call setup by default (and not terminate)
+setup=1
+
+# Prometheus release default
+tag="v0.8.0"
```

Comment:
Have you tried the latest prometheus on kind?

Reply:
Yes, I tested with `v0.0.13`. It works locally with `kubernetes version: 1.28` but I haven't changed it since we have that github runner CPU restrictions.

Reply:
They have added this compatibility table on their github page. We can use that to run accordingly.
https://github.com/prometheus-operator/kube-prometheus?tab=readme-ov-file#compatibility